index.js

```diff
@@ -0,0 +1,103 @@
+/*
+# 1) Создать массив содержащий все типы данных которые мы изучили
+# - спрашиваем у пользователя 2 операнда (2 промта).
+# - первым будет добавляемый тип, вторым экшен для преобразования соответствующего типа.
+# - добавляем готовый результат в массив.
+# - в конце всех операций выводим получившийся массив в консоль.
+# - массив должен содержать все 8 типов данных JavaScript.
+#
+Вопрос: Первым операндом должен быть добавляемый тип. 
+Это значит, что пользователь непосредственно вводит слово типа "Number,
+ String, Boolean", или он вводит что угодно, 
+ а мы потом выполняем операции над ним?
+И второе, "вторым экшен для преобразования" - то есть во втором 
+промпте пользователь должен ввести действие?? ввести слово типа 
+"добавить, удалить"?? не совсем могу понять условие.....
+
+Ответ: Первый операнд подразумевает под собой данные например - «1234». 
+Суть задачи чтобы вы вводили определенные данные и обрабатывали 
+их определенной функцией (в моем описании это «экшен») 
+*/
+
+
+let arr = [];
+let currentMonth = String (prompt('Current Month?', 'May'));
```

Comment:
```suggestion
let currentMonth = String(prompt('Current Month?', 'May'));
```
Не обязательно использовать `String` так как `prompt` и так возвращает строку.